docs/dataset-apc.md

```diff
@@ -20,6 +21,214 @@ Essentially, a spell is a collection of episodes (often just one episode) denoti
+```
+```
+# all normal admissions excluding transfers, births and maternity admissions
+with_admission_method = ['11', '12', '13', '21', '2A', '22', '23', '24', '25', '2D'] 
+```
+
+#### Brief Reference table
+Codes | Admission method
+-- | -- 
+11, 12, 13 | Elective (planned/booked/waiting list)
+21 | Emergency Admission: from A&E | usually include
+2A | Emergency Admission: from A&E of another provider
+22, 23, 24, 25, 2D | Emergency Admission: other sources (e.g. GP, MH crisis team)
+28 | Emergency Admission: Other - NB can include transfers and births
+2B | Transfer (emergency) 
+81 | Transfer (non-emergency)
+2C, 82, 83 | Birth of baby (in hospital or e.g. at home)
+31, 32 | Maternity Admission: Mother admitted ante or post partum
+
+
+### `source_of_admission`
+#### Filtering 
+- Filter using `with_source_of_admission=` (supply either an individual code or a list)
+#### Returning
+- `returning="source_of_admission"`
+#### Most useful for:
+  - identifying patients admitted from care homes (`'54', '65', '85', '88'`)
+  - excluding birth spells (`'79'`)
+  - identifying transfers* - but this cannot distinguish transfers from wards vs patients arriving from the A&E dept of another provider (e.g. an MIU or smaller hospital which doesn't have a suitable ward) so `admission_method` is better for this purpose. 
+
+#### Example
+```
+with_source_of_admission = ['19','29','54', '65', '85', '88', '39', '40', '41', '42', '66', '49', '52', '53', '87', '98', '99']
+# all admissions except births (includes unknowns (98, 99))
+```
+
+#### Brief Reference table
+code | Source of admission
+-- | --
+19 | Usual place of residence or no fixed abode (but should exclude care homes)
+29 | Temporary place of residence (e.g. hotel, school)
+54, 65, 85, 88 | Care Home / hospice
+51 | NHS other Hospital Provider - ordinary (physical care) ward or A&E
+49, 52, 53 | NHS other Hospital Provider - psychiatric accommodation, maternity ward/neonates or MH/LD ward
+39, 40, 41, 42, 66 | Other (Penal establishment, Court, or Police Station / Police Custody Suite,  foster care)
+79 | Babies born in or on the way to hospital
+87 | Non NHS run hospital
+98, 99 | Default/unknown
+
+
+### `patient_classification`
+
+#### Most useful for:
+- excluding regular admissions (i.e. where patient is admitted e.g. every week for chemotherapy or dialysis) 
+- excluding mother and baby admissions (where using delivery facilities only)
+- distinguishing elective day case admissions from ordinary admissions (day case admissions are for planned procedures not requiring overnight stays)
+
+An advantage of this field is that it reflects what actually happened during the spell as well as what was planned to occur - if treatment was not limited to the planned course (e.g. patient was expected to have their routine chemotherapy but e.g. illness arose which required further treatment and an overnight stay), this value defaults to `1`. 
```

Comment:
> This value defaults to `1`. 

Does this mean that if `patient_classification` is not specified, then only ordinary admissions are picked up? Has this always been the case?

Reply:
No this refers to the coding, not the cohortextractor - will clarify